### PR TITLE
Hide error 'message' when not used 

### LINKED ISSUE #542: Review errors show  "en.error.required.message" string
**What did you do?**

On a partially filled out Introduction section, click the Review sub-section menu.

**What did you expect to see?**

Full list of validation errors, with meaningful error messages.

**What did you see instead?**

For many of the error messages I saw: "Your response is required" followed by "en.error.required.message"

error message

**Additional Information**

Believe this is a regression on the `develop` branch.  Seen with latest Chrome on macOS 10.12 and IE11 on Windows 8.1.

### PR DESCRIPTION
**Pull request by line47**
Fixes #542

### CODE REVIEW

File: src/components/Form/Accordion/Accordion.scss [hunk 1]
```diff
@@ -148,6 +148,10 @@ $breakpoints: (
       border: 2px solid $eapp-red-light;
       border-radius: 6px;
       max-width: 64rem;
+      line-height: 2.5rem;
+      p {
+        margin-bottom: 0;
```
Comment (line47) [inline src/components/Form/Accordion/Accordion.scss]: Added to fix error message spacing for accordion items

File: src/components/Form/Field/Field.jsx [hunk 2]
```diff
@@ -13,10 +13,18 @@ const message = (id) => {
...
   }
 
+  const messageId = `${id}.message`
+  let message = i18n.m(messageId)
+  if (Object.prototype.toString.call(message) === '[object String]' && message.indexOf(messageId) > -1) {
+    message = ''
+  } else {
+    message = <span>{message}</span>
+  }
+
```
Comment (line47) [inline src/components/Form/Field/Field.jsx]: This could probably be simplified some but it works ATM

File: src/components/Form/Field/Field.scss [hunk 3]
```diff
@@ -254,7 +254,6 @@
           text-align: center;
           background: $eapp-blue-light;
           border-radius: 50%;
-          padding-right: 0.1rem;
         }
 
         em {
```
Comment (line47) [inline src/components/Form/Field/Field.scss]: ☝️ Fixing more spacing issues with error messages 😅

Comment (ryanhofdotgov) [reply src/components/Form/Field/Field.scss]: Just double-checking, but do these CSS changes work well for all 3 cases?
* when there is title, message, and note
* when there is title, message
* when there is just title

Comment (line47) [reply src/components/Form/Field/Field.scss]: Yeah the looks good 👍 

screen shot 2018-07-13 at 8 59 34 am
screen shot 2018-07-13 at 8 56 36 am
screen shot 2018-07-13 at 8 56 44 am

**Pull request merged**